src/Services/SearchService.php

```diff
@@ -247,6 +259,34 @@ public function handleProductRedirectUrl(string $url)
         header('Location: ' . $url);
     }
 
+    /**
+     * Since Ceres version 5.0.3 invalid ids are filtered by default.
+     * Custom version checking as Plentymarkets forbids using the version_compare function
```

Comment:
The sentence `Custom version checking as Plentymarkets forbids using the version_compare function`  is implementation specific, therefore please put it as inline doc before declaring the `$versionParts` variable.